website/docs/cli/commands/auth/tutorials/migrating-from-leapp.mdx

```diff
@@ -0,0 +1,259 @@
+
+### Equivalent `atmos auth` Configuration
+
+```yaml
+auth:
+  providers:
+    acme-sso: # (1)
+      kind: aws/iam-identity-center
+      region: us-east-2 # (5)
+      start_url: https://acme.awsapps.com/start/ # (1.a)
+
+  identities:
+    acme-identity: # (4)
+      kind: aws/permission-set
+      via:
+        provider: acme-sso
+      principal:
+        name: "IdentityManagersTeamAccess" # (3)
+        account:
+          name: "core-identity" # (2)
+```
+
+## Field Mapping Reference
+
+| Leapp Field | `atmos auth` Location | Notes |
+|-------------|----------------------|-------|
+| (1) Provider name | `providers.<name>` | Use a descriptive name (e.g., `acme-sso`) |
+| (1.a) Start URL | `providers.<name>.start_url` | Found in AWS IAM Identity Center portal |
+| (5) Region | `providers.<name>.region` | From Leapp session's Region column |
+| (2) Session (account) | `identities.<name>.principal.account.name` | AWS account name |
+| (3) Identity (permission set) | `identities.<name>.principal.name` | Exact permission set name |
+| (4) Named Profile | `identities.<name>` (key) | Use descriptive naming: `account/role` |
+
+## Step-by-Step Migration
+
+### 1. Identify Your Provider Configuration
+
+In Leapp, find your provider in the left sidebar under "Integrations".
+
+**Create the provider in `atmos.yaml`:**
+
+```yaml
+auth:
+  providers:
+    <provider-name>:
+      kind: aws/iam-identity-center
+      region: <region>
+      start_url: <your-sso-start-url>
+```
+
+**Example:**
+```yaml
+auth:
+  providers:
+    acme-sso:
+      kind: aws/iam-identity-center
+      region: us-east-2
+      start_url: https://acme.awsapps.com/start/
+```
+
+### 2. Convert Each Session to an Identity
+
+For each session in Leapp you use, create an identity configuration:
+
+```yaml
+auth:
+  identities:
+    <account-name>/<descriptive-role-name>:
+      kind: aws/permission-set
+      via:
+        provider: <provider-name>
+      principal:
+        name: "<PermissionSetName>"
+        account:
+          name: "<account-name>"
+```
+
+**Example:**
+```yaml
+auth:
+  identities:
+    acme-identity:
+      kind: aws/permission-set
+      via:
+        provider: acme-sso
+      principal:
+        name: "IdentityManagersTeamAccess"
+        account:
+          name: "core-identity"
+```
+
+### 3. Set a Default Identity (Optional)
+
+Mark one identity as default for convenience:
+
+```yaml
+auth:
+  identities:
+    acme-identity:
+      default: true
+      kind: aws/permission-set
+      via:
+        provider: acme-sso
+      principal:
+        name: "IdentityManagersTeamAccess"
+        account:
+          name: "core-identity"
+```
+
+### 4. Test Your Configuration
+
+```bash
+# Validate configuration
+atmos auth validate
+
+# Login with default identity
+atmos auth login
+
+# Check authentication status
+atmos auth whoami
+
+# Use with Terraform
+atmos terraform plan <component> -s <stack>
+```
+
+## Integration with Geodesic
```

Comment:
Is this strictly for migrating from Leapp for Geodesic? Or is this more a tutorial for Geodesic?